vars/skipStage.groovy

```diff
@@ -420,6 +420,18 @@ boolean call(Map config = [:]) {
         case "Bullseye Report on EL 8":
             return env.BULLSEYE == null ||
                    skip_stage_pragma('bullseye', 'true')
+        case "Functional Hardware Small TCP":
+            return ! paramsValue('CI_FUNCTIONAL_HARDWARE_SMALL_TCP', true)
```

Comment:
We're starting to write Groovy according to coding conventions and standards.  We don't have a linting stage in CI yet, but I have a linter in VS Code and I know that it will tell me that there is not supposed to be a space after the `!` operator.

Reply:
I can change this for the new code, but I just copied what was already established in the file so there will be a large number of offenses.

Reply:
Done